docker-compose.yml

```diff
@@ -43,6 +62,7 @@ services:
     volumes:
       - ./app-backend/:/opt/raster-foundry/app-backend/
       - ./.sbt:/root/.sbt
+      - ./worker-tasks/:/opt/raster-foundry/worker-tasks/
```

Comment:
This is pointing to Spark testing stuff. I think removing it from here is fine. It was likely added to test something out.